bin/utils/bulk_compression.py

```diff
@@ -0,0 +1,416 @@
+# PCSX2 - PS2 Emulator for PCs
+# Copyright (C) 2024 PCSX2 Dev Team
+#
+# PCSX2 is free software: you can redistribute it and/or modify it under the terms
+# of the GNU Lesser General Public License as published by the Free Software Found-
+# ation, either version 3 of the License, or (at your option) any later version.
+#
+# PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with PCSX2.
+# If not, see <https://www.gnu.org/licenses/>.
+
+import sys
+import os
+import re
+from subprocess import Popen, PIPE
+from os.path import exists
+
+gamecount = [0]
+
+# =================================================================================================
+
+def deletionChoice(file_type):                                      # Choose to delete source files
+    yesno = {
+        "n"   : 0,
+        "no"  : 0,
+        "y"   : 1,
+        "yes" : 1,
+    }
+    print("║                                                                                ")
+    print("║                                                                                ")
+    print(f"║ Do you want to delete the original {file_type.upper()} files as they are converted?")
+    choice = input("║ Type Y or N then press ENTER: ")
```

Comment:
`lower()` doesn't modify the string instance, it returns a lowercase string. This made `N`, `YeS`, etc not work.

You can remove the `choice.lower()` and just amend the input call.

```suggestion
choice = input("║ Type Y or N then press ENTER: ").lower()
```

Reply:
Ah, yup, I meant to set choice = choice.lower(), but this is a better solution. Done.